draft-birkholz-rats-epoch-markers.md

```diff
@@ -206,7 +206,13 @@ In a highly available service (e.g., a cloud attestation verifier) having to
 keep per-session nonce state poses scalablity problems.  An alternative is to
 use time-synchronised servers that share a symmetric key, which produce and
 consume nonces based on coarse-grained clock ticks that are signed using the
-shared secret.
+shared secret.  This way, a nonce minted by a server in the pool can be
+processed by any other server in pool, which avoids the need for session
+"stickiness."
+
+A stateless-nonce supports the above use case by encoding a Posix time (i.e.,
+the epoch identifier), alongside a minimal set of metadata, authenticated with
+a symmetric key in a self-contained and compact (starting at 40 bytes) token.
```

Comment:
The "starting at" is not Bourne out by the CDDL.

Reply:
Not sure I get your point: with a 0-length Pad (the inf), it serialises to 40 bytes.  That's what I wanted to say here.

Reply:
I didn't see how to go beyond the 40-byte token, but I probably focused on the non-agile hash too much.

Reply:
ok, got it, sorry, not enough coffee :-)